src/resources/LibraryStyles.css

```diff
@@ -199,56 +199,58 @@ input {
 }
 
 .SearchBar {
-    background: #5b5a5a;
     width: 100%;
+    background: #5b5a5a;
 }
 
 .SearchInput {
-    width: 100%;
-    height: 40px;
     display: flex;
     justify-content: space-around;
-    align-items: center;
+    padding: 5px 0px;
 }
 
 .SearchInputContainer {
     width: 100%;
-    height: 30px;
     background-color: #fff;
     display: flex;
     justify-content: space-around;
     align-items: center;
 }
 
 .SearchBar .SearchBarIcon {
-    margin-left: 10px;
+    padding-left: 10px;
 }
 
 .SearchBar #SearchInputText {
-    height: 18px;
     width: 100%;
     font-size: 15px;
-    padding: 15px 10px 10px 10px;
+    padding: 5px 5px 0px 10px;
     background: none;
     outline: none;
     border: none;
 }
 
-.SearchBar #SearchOptionsBtn {
-    height: 32px;
-    width: 32px;
-    color: #ddd;
-    border: 0px;
+.SearchBar #SearchOptionsBtnEnabled,
+.SearchBar #SearchOptionsBtnDisabled {
+    height: 30px;
+    width: 30px;
+    border: none;
+    outline: none;
     background-color: #414141;
 }
 
-.SearchBar #SearchOptionsBtn:hover {
-    background-color: rgba(255, 255, 255, 0.1);
-    cursor: pointer;
```

Comment:
I have a thought about making this consistent when we still can, that is, remove element identifier unless there's a need to identify them by their IDs. Can we have a task to standardise all to use `className` instead?